src/main/java/org/embulk/input/zendesk/services/ZendeskUserEventService.java

```diff
@@ -0,0 +1,158 @@
+    {
+        return new ObjectMapper().createObjectNode().set(task.getTarget().getJsonName(), mockJsonNode());
+    }
+
+    private String buildOrganizationURI()
+    {
+        return ZendeskUtils.getURIBuilder(task.getLoginUrl())
+                .setPath(ZendeskConstants.Url.API + "/" + Target.ORGANIZATIONS.getJsonName())
+                .setParameter("per_page", "100")
+                .setParameter("page", "1")
+                .toString();
+    }
+
+    private String buildOrganizationWithUserURI(final String path)
+    {
+        return path.replace(".json", "")
+                + "/users.json?per_page=100&page=1";
+    }
+
+    private String buildUserEventURI(final String userID)
+    {
+        final URIBuilder uriBuilder = ZendeskUtils.getURIBuilder(task.getLoginUrl())
+                .setPath(ZendeskConstants.Url.API_USER_EVENT)
+                .setParameter("identifier", task.getProfileSource().get() + ":user_id:" + userID);
+
+        task.getUserEventSource().ifPresent(eventSource -> uriBuilder.setParameter("source", eventSource));
+        task.getUserEventType().ifPresent(eventType -> uriBuilder.setParameter("type", eventType));
+        task.getStartTime().ifPresent(startTime -> {
+            try {
+                uriBuilder.setParameter("start_time", ZendeskDateUtils.convertToDateTimeFormat(startTime, ZendeskConstants.Misc.ISO_INSTANT));
+            }
+            catch (DataException e) {
+                uriBuilder.setParameter("start_time", ZendeskDateUtils.convertToDateTimeFormat(Instant.EPOCH.toString(), ZendeskConstants.Misc.ISO_INSTANT));
+            }
+        });
+
+        task.getEndTime().ifPresent(endTime -> uriBuilder.setParameter("end_time", ZendeskDateUtils.convertToDateTimeFormat(endTime, ZendeskConstants.Misc.ISO_INSTANT)));
+
+        return uriBuilder.toString();
+    }
+
+    private JsonNode mockJsonNode()
+    {
+        try {
+            String mockData = "[\n" +
+                    "    {\n" +
+                    "      \"id\": \"5c7f31aef8df240001e60bbf\",\n" +
+                    "      \"type\": \"remove_from_cart\",\n" +
+                    "      \"source\": \"shopify\",\n" +
+                    "      \"description\": \"\",\n" +
+                    "      \"authenticated\": true,\n" +
+                    "      \"created_at\": \"2019-03-06T02:34:22Z\",\n" +
+                    "      \"received_at\": \"2019-03-06T02:34:22Z\",\n" +
+                    "      \"properties\": {\n" +
+                    "        \"model\": 221,\n" +
+                    "        \"size\": 6\n" +
+                    "      },\n" +
+                    "      \"user_id\": \"12312354234\"\n" +
+                    "    }\n" +
+                    "]";
+
+            return new ObjectMapper().readTree(mockData);
+        }
+        catch (IOException ex) {
+            throw new RuntimeException("Error when mock data for guess or preview " + ex.getMessage());
```

Comment:
I'd suggest just said `Can not create sample data {ex.getMessage()}`

Reply:
fixed